Dockerfile

```diff
@@ -13,49 +15,50 @@ ENV TAUTULLI_DOCKER=True
 RUN \
   echo "**** install build packages ****" && \
   apk add --no-cache --virtual=build-dependencies \
-    g++ \
-    gcc \
-    make \
+    build-base \
+    cargo \
     python3-dev && \
   echo "**** install packages ****" && \
   apk add --no-cache \
-    curl \
-    jq \
-    py3-openssl \
-    py3-pip \
-    py3-setuptools \
+    git \
     python3 && \
- echo "**** install pip packages ****" && \
-  python3 -m pip install --upgrade pip wheel && \
-  pip install -U --no-cache-dir --find-links https://wheel-index.linuxserver.io/alpine-3.16/ \
-    mock \
-    plexapi \
-    pycryptodomex && \
-  echo "**** install app ****" && \
-  mkdir -p /app/tautulli && \
+  echo "**** install tautulli ****" && \
   if [ -z ${TAUTULLI_RELEASE+x} ]; then \
     TAUTULLI_RELEASE=$(curl -sX GET "https://api.github.com/repos/Tautulli/Tautulli/releases/latest" \
     | jq -r '. | .tag_name'); \
   fi && \
+  mkdir -p /app/tautulli && \
   curl -o \
-  /tmp/tautulli.tar.gz -L \
+    /tmp/tautulli.tar.gz -L \
     "https://github.com/Tautulli/Tautulli/archive/${TAUTULLI_RELEASE}.tar.gz" && \
   tar xf \
-  /tmp/tautulli.tar.gz -C \
+    /tmp/tautulli.tar.gz -C \
     /app/tautulli --strip-components=1 && \
+  cd /app/tautulli && \
+  sed -i 's/^backports.zoneinfo==0.2.1$/backports.zoneinfo==0.2.1;python_version<"3.9"/' requirements.txt && \
```

Comment:
This will not be necessary if https://github.com/Tautulli/Tautulli/pull/1926 merges and eventually is included in a stable release.